tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -4274,6 +4274,202 @@ TEST (nnstreamer_capi_singleshot, set_input_info_success_02)
   ml_tensors_info_destroy (out_res);
 }
 
+/**
+ * @brief Test NNStreamer single shot (tflite)
+ * @detail run the `ml_single_invoke_dynamic` api works properly.
+ */
+TEST (nnstreamer_capi_singleshot, invoke_dynamic_success)
+{
+  ml_single_h single;
+  int status;
+  ml_tensors_info_h in_info, out_info;
+  ml_tensors_data_h input, output;
+
+  unsigned int tmp_count;
+  ml_tensor_type_e tmp_type = ML_TENSOR_TYPE_UNKNOWN;
+  ml_tensor_dimension tmp_dim;
+
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
+  gchar *test_model;
+
+  /* supposed to run test in build directory */
+  if (root_path == NULL)
+    root_path = "..";
+
+  /* dynamic dimension supported */
+  test_model = g_build_filename (root_path, "tests", "test_models", "models",
+      "add.tflite", NULL);
+  ASSERT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS));
+
+  status = ml_single_open (&single, test_model, NULL, NULL,
+      ML_NNFW_TYPE_TENSORFLOW_LITE, ML_NNFW_HW_ANY);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_single_get_input_info (single, &in_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_tensors_data_create (in_info, &input);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  float tmp_input[] = {1.0};
+  float * output_buf;
+  size_t data_size;
+  status = ml_tensors_data_set_tensor_data(input, 0, tmp_input,
```

Comment:
minor convention issues: how about add a space like  `... set_tensor_data (input, 0, tmp_input, ...` here and other lines

Reply:
gst-indent has been applied